test/otp/utils.go

```
@@ -0,0 +1,693 @@
+	})
+	assertWaitPollNoErr(err, fmt.Sprintf("Failed to create Admin Network Policy CR %v", anp.name))
+}
+
+func applyResourceFromTemplateByAdmin(oc *ocCLI, parameters ...string) error {
+	var configFile string
+	err := wait.Poll(3*time.Second, 60*time.Second, func() (bool, error) {
+		tmpFile, err := os.CreateTemp("", getRandomString()+"resource-*.json")
+		if err != nil {
+			return false, nil
+		}
+		tmpFile.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `tmpFile.Close` is not checked

(errcheck)